networkx/algorithms/bridges.py

```diff
@@ -58,7 +59,13 @@ def bridges(G, root=None):
     n)`, ignoring polylogarithmic factors, where *n* is the number of
     nodes in the graph and *m* is the number of edges.
 
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Bridge_(graph_theory)#Bridge-Finding_with_Chain_Decompositions
     """
+    # This is an implementation of the algorithm described in _[1]. Bridges can
+    # be found using chain decomposition. An edge e in G is a bridge if and
+    # only if e is not contained in any chain.
     chains = nx.chain_decomposition(G, root=root)
```

Comment:
I had my own implementation of this algorithm, but in lieu of #2525 I used that implementation. I merged in the documentation from my own function because I think its helpful. Not sure if that's appropriate for this PR.